.github/workflows/compliance.yml

```diff
@@ -0,0 +1,49 @@
+name: Run compliance tests
+
+on:
+  workflow_dispatch:
```

Comment:
Actually, test validation should run whenever this repo is changed. I'm not sure we should trigger a run when the test runner repo is changed. Actually, rather not, because those are different responsibilities. We can't know about _all_ people writing OpenAPI tests and send triggers to run their CIs.

If anything, we can run periodic tests to check whether the latest runner version still validates our tests. But I think we can take care of this later.

So rather, instead of `workflow_dispatch`, I would trigger the run on PRs and pushes, as usual.